src/Unzipper.cpp

```diff
@@ -10,10 +10,10 @@
 #include "utils/Path.hpp"
 #include "utils/glob.hpp"
 
-#include "external/minizip/ioapi_mem.h"
-#include "external/minizip/minishared.h"
-#include "external/minizip/unzip.h"
-#include "external/minizip/zip.h"
+#include <minizip/ioapi_mem.h>
+#include <minizip/minishared.h>
+#include <minizip/unzip.h>
+#include <minizip/zip.h>
```

Comment:
Well, potatoes. Can you fix the Makefile part to account for this change? This change would be needed to support using system packages.

Reply:
This is not an issue for me with Makefile: it's just fixing a -I piece of cake. I find sad that the hard part is always coming from CMake. On the original project 99% of issues come from CMake that is why I used my Makefile system. I'm not even able to compile with your changes. The nightmare is coming back again ;(

Reply:
There are not optional packages. They either comes from initially with `make download-external-libs` then I added git submodules for Red-Hat

Reply:
> -- Configuring done (10.5s)
> CMake Error: install(EXPORT "zipperTargets" ...) includes target "zipper" which requires target "zlib" that is not in any export set.

This is a complex issue about static libraries requiring to have all dependencies installed. Iirc, that is an existing issue.

I can look into that afterwards, but for now, try building with `-DZIPPER_SHARED_LIBS=ON`

Reply:
> I'm not sure to switch "" to <> because, if I remember well the compiler will search for OS files first

Yes, but that is desired. You are signalling that this is a third-party library. The `-I` flags will make sure that the system dependency is picked up.

> @LecrisUT why not letting people doing git clone ... --recurse then either our CMakeLists includes directly the CMake inside the external/minizip/CMakeLists.txt our we compile the C++ files directly ? I can modify headers for you now

The reason for this change is to make way to allow building against a system `minizip` which exports it as `#include <minizip/*>`. This will make it easier to pull in `minizip-ng-compat-devel` make it compatible with that, and move the bundled `minizip` to the upstream version.